csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/FileContent.cs

```diff
@@ -164,6 +185,12 @@ private void DoInitialize()
                                 implicitUsingNamespaces.Add(ns);
                             }
                         }
+
+                        // Determine project structure:
+                        isLegacyProjectStructureUsed |= MicrosoftCSharpTargets().IsMatch(line);
```

Comment:
`|=` uses logical or `|`, which does not have short circuit semantics (and unfortunately there doesn't exist `||=`).
You are also more than welcome to change the logic for `useAspNetCoreDlls` as well.

Reply:
I'm leaving the multi-line variants, such as `useAspNetCoreDlls`, as they are.